vnext/Shared/Networking/OriginPolicyHttpFilter.cpp

```diff
@@ -684,6 +683,38 @@ ResponseOperation OriginPolicyHttpFilter::SendPreflightAsync(HttpRequestMessage
   co_return {co_await m_innerFilter.SendRequestAsync(preflightRequest)};
 }
 
+#pragma region IRedirectEventSource
+
+bool OriginPolicyHttpFilter::OnRedirecting(
+    HttpRequestMessage const &request,
+    HttpResponseMessage const &response) noexcept {
+  // Consider the following scenario.
```

Comment:
>// Consi [](http://example.com/codeflow?start=2&length=8)

If there is an external article for this would you mind referencing this so we know this is standard practice to implement? 
It still feels weird that we have to implement this and that there is no library in eiterh windows  or shared that implements this.
This seems so frought with perril if everyone needs to implement this....

Reply:
There is none to mi knowledge.

The same algorithms are implemented in the Office variant, extending the underlying HTTP library in a similar fashion.